microsoft/testsuites/core/cpu.py

```diff
@@ -1,26 +1,98 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
+from __future__ import annotations
 
+import re
+import time
+from dataclasses import dataclass
 from pathlib import PurePosixPath
+from typing import List
 
 from assertpy.assertpy import assert_that
 
 from lisa import (
     BadEnvironmentStateException,
     LisaException,
     Logger,
-    Node,
     SkippedException,
     testsuite,
 )
-from lisa.tools import Cat, Echo, Lscpu, Lsvmbus, Uname
+from lisa.node import Node
```

Comment:
Node is at lisa level too.

Reply:
seems like auto-import feature in VS code is importing from the files directly.